bar-trender-front-end/src/components/ApiLoginForm.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+
+class POSTLoginForm extends React.Component {
+
+  constructor() {
+    super();
+
+    this.state = {
+      login: {
+        email: "client1@email.com",
+        password: "holamundo."
+      },
+      input: {},
+      errors: {}
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleLogin = this.handleLogin.bind(this);
+  }
+
+  handleLogin() {
+    alert('A form was submitted: ' + JSON.stringify(this.state));
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7123646)
```suggestion
    alert("A form was submitted: " + JSON.stringify(this.state));
```